src/components/NavMenuItems.tsx

```diff
@@ -1,59 +1,127 @@
 import React from 'react';
-import { Menu, Button } from 'semantic-ui-react';
-import { NavLink } from 'react-router-dom';
 import { PathURL, ExternalURL } from './Router';
-import { ISidebarClose } from './Nav';
-
-export function NavMenuItems(props: ISidebarClose) {
-  return (
-    <>
-      <Menu.Item
-        as={NavLink}
-        exact
-        to={PathURL.HOME}
-        position="right"
-        activeClassName="active"
-        className="navItem"
-        onClick={props.hideSidebar}
-      >
-        Home
-      </Menu.Item>
-
-      <Menu.Item
-        as={NavLink}
-        exact
-        to={PathURL.ABOUT}
-        activeClassName="active"
-        className="navItem"
-        onClick={props.hideSidebar}
-      >
-        About
-      </Menu.Item>
-
-      <Menu.Item
-        as={NavLink}
-        exact
-        to={PathURL.VOLUNTEER}
-        activeClassName="active"
-        className="navItem"
-        onClick={props.hideSidebar}
-      >
-        Volunteer
-      </Menu.Item>
-
-      <Menu.Item className="navItem">
-        <Button
-          as="a"
-          href={ExternalURL.OPEN_COLLECTIVE}
-          circular
-          basic
-          color="red"
-          size="mini"
-          onClick={props.hideSidebar}
-        >
-          <strong>Donate</strong>
-        </Button>
-      </Menu.Item>
-    </>
-  );
+import { NavHashLink } from 'react-router-hash-link';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faBars } from '@fortawesome/free-solid-svg-icons';
+
+export class NavMenuItems extends React.Component {
```

Comment:
We shouldn't need to turn this into a class component to add life cycle events. 
React hooks is a better API for this sort of thing. [Checkout this guide.](https://reactjs.org/docs/hooks-intro.html) The video by abramov is how i learned to use them. Highly recommend it.

Reply:
Thanks for the suggestion, implemented 👍